src/fernet/core.clj

```diff
@@ -85,8 +85,19 @@
     (catch Exception e
       (invalid-token))))
 
+(defn decrypt [key token & options]
+  "Decrypt the token and return the contents as a byte-array"
+  (apply decrypt-token key token options))
+
+(defn decrypt-to-string
+  "Decrypt the token and return the message as a string"
+  [key token & options]
+  (String. (apply decrypt-token key token options)))
+
 (defn encrypt [key message]
```

Comment:
Keep in mind that primitive-annotated functions are extra special on the JVM:

https://www.lvh.io/posts/what-the-heck-is-a-clojurelangifnlo.html